content/docs/get-started/aws/review-project.md

```diff
@@ -82,18 +82,33 @@ class Program
 This Pulumi program creates an S3 bucket and exports the name of the bucket.
 
 {{% lang python %}}
+
+> *Note*: As a prerequisite, [install virtualenv](https://virtualenv.pypa.io/en/latest/installation/) to manage project requirements
+
 For Python, before we deploy the stack, the following commands need to be run to create a virtual environment, activate it, and install dependencies:
 
+Create a virtual environment:
+
 ```bash
-$ virtualenv -p python3 venv
+$ python -m venv aws-env
 ```
 
+Activate the environment in Linux and MacOS:
+
 ```bash
-$ source venv/bin/activate
+$ source aws-env/bin/activate
 ```
 
+Activate the environment in Windows:
+
+```bash
+> aws-env\Scripts\activate.bat
```

Comment:
It'd be best to not include the `>` in front of `bat` snippets right now, because currently when our copy button for code snippets is clicked, it won't strip the `>` from the text that is added to the clipboard like we do for `$` in `bash`/`shell`/`sh` snippets.

https://github.com/pulumi/docs/blob/a3a71548eee00c5fddaee66e9154aee5b364e24b/assets/js/copybutton.js#L17-L22

Reply:
accepted